src/Microsoft.Framework.CopyOnWriteDictionary.Internal/project.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.0.0-*",
+    "shared": "*.cs",
+    "dependencies": {
```

Comment:
https://github.com/aspnet/XRE/issues/1237 - Adding the dependency currently results in duplicate code errors in referrencing projects (if they reference more than 1 shared source assembly). Pretty horrible to have to revert to this but this way we don't end up breaking users when aspnet/XRE#1237 is completed.